test/test_torch.py

```diff
@@ -3328,6 +3328,32 @@ def test_pickle_function(self):
         b = pickle.loads(serialized)
         self.assertEqual(a, b)
 
+    def test_pickle_gradscaler(self):
```

Comment:
can you add it to some of the device-generic tests rather than _TorchMixin? We are trying to get rid of the latter.

Reply:
Moved test to TestTorchDeviceType, and made it more devoutly device agnostic (see comments in code)